xml/deploy_cephadm.xml

```diff
@@ -758,6 +796,7 @@ o- / ............................................................... [...]
   | o- advanced .................................................... [...]
   | o- ceph_conf ................................................... [...]
   | o- dashboard ................................................... [...]
+  |   o- force_password_update ................................. [enabled]
```

Comment:
Note that I also update this screen output in #409 - we also added two more options to configure the dashboard's SSL certificate and key via `ceph-salt`.

Reply:
We'll deal with the merge conflict when it comes to it. It shouldn't be much. I just need @sebastian-philipp to review this PR, and then we can merge.